usaon_vta_survey/templates/response/base.html

```diff
@@ -3,7 +3,9 @@
 
 {% block content %}
 
-  <h2>{% block title %}Response: {{survey.id}}{% endblock %}</h1>
+  <h2>{% block title %}Response to: {{survey.title}}{% endblock %}</h1>
+  <h3>Survey ID: {{survey.id}}</h3>
```

Comment:
Maybe even smaller? This could even be plain paragraph text. Since users don't really care about this value, I think we should keep it as out-of-the-way as we can.